test/helpers/actions.ts

```diff
@@ -347,8 +347,7 @@ export async function swipeFullScreen(direction: Direction) {
       actions: [
         { type: 'pointerMove', duration: 0, x: startX, y: startY },
         { type: 'pointerDown', button: 0 },
-        { type: 'pause', duration: 50 },
-        { type: 'pointerMove', duration: 200, x: endX, y: endY },
+        { type: 'pointerMove', duration: 100, x: endX, y: endY },
```

Comment:
👎 
The param values are tailored to work reliably on both platforms. I think it may cause flakiness.